src/App/version_txt.py

```diff
@@ -10,22 +10,54 @@
 # FOR A PARTICULAR PURPOSE
 #
 ##############################################################################
+import collections
+import re
 import sys
 
 import pkg_resources
 
 _version_string = None
+_zope_version = None
+
+ZopeVersion = collections.namedtuple(
+    "ZopeVersion",
+    ["major", "minor", "micro", "status", "release"]
+    )
 
 
 def _prep_version_data():
-    global _version_string
+    global _version_string, _zope_version
     if _version_string is None:
         v = sys.version_info
         pyver = "python %d.%d.%d, %s" % (v[0], v[1], v[2], sys.platform)
         dist = pkg_resources.get_distribution('Zope')
         _version_string = "%s, %s" % (dist.version, pyver)
 
+        expr = re.compile(
+            r'(?P<major>[0-9]+)\.(?P<minor>[0-9]+)(\.(?P<micro>[0-9]+))?'
+            '(?P<status>[A-Za-z]+)?(?P<release>[0-9]+)?')
+        dict = expr.match(dist.version).groupdict()
```

Comment:
Can we avoid the use of the name dict for a variable?

Reply:
`dict` renamed to `version_dict`.
Added `getZopeVersion` related entry to `CHANGES.rst`.
Added tests for `getZopeVersion`: Note that 2 unrelated tests from `test_ApplicationManager` failed with `AttributeError: module 'Testing' has no attribute 'testbrowser'`(maybe due to an outdated `Testing` module).